frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -73,15 +78,28 @@ export default function ViewFacility() {
 
   useEffect(() => {
     if (bookingStatus === BookingStatus.SUCCESS) {
-      dispatch(setBookingStatus(BookingStatus.INITIAL))
+      async function wait3seconds() {
+        const promise = new Promise((resolve, reject) => {
+          setTimeout(() => resolve(dispatch(setBookingStatus(BookingStatus.INITIAL))), 3000)
+        }) // wait 3 seconds before setting booking status back to initial
+        const result = await promise
+        result
+      }
       console.log('sucessssssssss')
+      wait3seconds()
```

Comment:
ok firstly well done! the function works just as intended. but here are some feedback on your code:
1. There's no need for us to collect `result` from the async func sine we are not using it. So we can just write `await new Promise` in one line
2. Name your function a more meaningful name so that people can understand what the function does just by reading it.
3. For functions, you can use the [JSDoc block comment](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) to better document functions. (I'll be briefing y'all on how to do documentation soon, but good to take note for now)
4. Abstracted the await time so that it is easier to modify in the future, and clearer as well

Reply:
Do read my comments, and commit the change if you deem fit